apps/dotcms-ui/src/app/view/components/dot-content-compare/store/dot-content-compare.store.ts

```diff
@@ -147,8 +154,17 @@ export class DotContentCompareStore extends ComponentStore<DotContentCompareStat
         return contents;
     }
 
-    //Effects
+    bringBack(inode: string): Observable<DotVersionable> {
+        return this.dotVersionableService.bringBack(inode).pipe(
+            take(1),
+            map((version: DotVersionable) => {
+                this.dotRouterService.goToURL(`/c/content/${version.inode}`);
+                return version;
+            })
+        );
+    }
```

Comment:
This is not modifying the state, so it should not be here, in this case should live in the component.